armadillo/src/main/java/org/molgenis/armadillo/storage/ArmadilloStorageService.java

```diff
@@ -133,4 +200,31 @@ public InputStream loadSystemFile(String name) {
       return InputStream.nullInputStream();
     }
   }
+
+  private void throwIfDuplicate(String project, String object) {
```

Comment:
why do you use this intermediate function instead of throwing the exception directly at each spot?

Reply:
Ah yes, I should have looked better.